README.md

```diff
@@ -35,37 +43,57 @@ To make your kernel compatible with `bootloader`:
     };
     bootloader_api::entry_point!(kernel_main, config = &CONFIG);
     ```
-- Compile your kernel to an ELF executable by running **`cargo build --target x86_64-unknown-none`**. You might need to run `rustup target add x86_64-unknown-none` before to download precompiled versions of the `core` and `alloc` crates.
+- Compile your kernel to an ELF executable by running **`cargo build --target x86_64-unknown-none`**. You might need to run `rustup target add x86_64-unknown-none` before to download precompiled versions of the `core` and `alloc` crates. You can add `x86_64-unknown-none` as default target and add it to your toolchain so that `cargo build` takes care of this.
+    ```toml
+    # .cargo/config.toml
+    [build]
+    target = "x86_64-unknown-none"
+    ```
+    ```sh
+    $ cargo build
+    ```
 - Thanks to the `entry_point` macro, the compiled executable contains a special section with metadata and the serialized config, which will enable the `bootloader` crate to load it.
 
-### Booting
-
-To combine your kernel with a bootloader and create a bootable disk image, follow these steps:
+#### Setup the workspace
```

Comment:
Only the first three steps are relevant for setting up a workspace.

How about "Creating a bootable image"?